wgsl/index.bs

```diff
@@ -8138,6 +8138,7 @@ A program [=shader-creation error|must=] satisfy the following limits:
     <tr><td>[=Nesting depth=] of a [=composite=] type<td>255
     <tr><td>Number of [=formal parameter|parameters=] for a function<td>255
     <tr><td>Number of case selector values in a [=statement/switch=] statement<td>16383
+    <tr><td>[=Byte-size=] of an [=array=] type<td>655in35
```

Comment:
How does this work for types that have no defined byte size? e.g. `bool` and abstract numerics?

Reply:
I think it'd be ok to leave that based on the implementation. We could add a second limit on number of elements if that is necessary.

Reply:
I suggest adding:

>   For this limit, assume bool has byte-size of 4.

Reply:
What about abstract numerics? Suggest 8 bytes for them, given that we guarantee at least 64 bit precision.